### PR TITLE
KT-26361 @Deprecated "ReplaceWith" quickfix inserts 'this' incorrectly when using function imports

### PR DESCRIPTION
**Pull request by t-kameyama**
https://youtrack.jetbrains.com/issue/KT-26361

### CODE REVIEW

File: idea/src/org/jetbrains/kotlin/idea/codeInliner/CodeToInlineBuilder.kt [hunk 1]
```diff
@@ -169,18 +170,19 @@ class CodeToInlineBuilder(
...
-                    val receiver = if (resolvedCall.resultingDescriptor.isExtension)
-                        resolvedCall.extensionReceiver
-                    else
-                        resolvedCall.dispatchReceiver
-                    if (receiver is ImplicitReceiver) {
-                        val resolutionScope = expression.getResolutionScope(bindingContext, resolutionFacade)
-                        val receiverExpression = receiver.asExpression(resolutionScope, psiFactory)
-                        if (receiverExpression != null) {
-                            receiversToAdd.add(Triple(expression, receiverExpression, receiver.type))
+                if (targetCallable !is FunctionImportedFromObject) {
```
Comment (dimonchik0036) [inline idea/src/org/jetbrains/kotlin/idea/codeInliner/CodeToInlineBuilder.kt]: `ImportedFromObjectCallableDescriptor<*>`

**Comment by t-kameyama** [comment]
Well, the test is successful in my environment.

スクリーンショット 2019-10-22 17 09 58

**Comment by dimonchik0036** [comment]
Failed after rebase to master.

**Comment by t-kameyama** [comment]
The test was successful in my environment after rebase to master (02fc921eedc).

スクリーンショット 2019-10-22 20 42 56

**Pull request closed without merging**